app/domain/sac_cas/beitragskategorie/calculator.rb

```diff
@@ -12,18 +12,18 @@ class Calculator
     CATEGORY_YOUTH = :youth
     CATEGORY_FAMILY = :family
     BEITRAGSKATEGORIEN = [CATEGORY_ADULT, CATEGORY_YOUTH, CATEGORY_FAMILY].map(&:to_s).freeze
-    
+
     AGE_RANGE_ADULT = 22..199
     AGE_RANGE_MINOR_FAMILY_MEMBER = 6..17
     AGE_RANGE_YOUTH = AGE_RANGE_MINOR_FAMILY_MEMBER.begin...AGE_RANGE_ADULT.begin
-    
+
     def initialize(person, reference_date: Time.zone.today)
       @person = person
       @reference_date = reference_date
     end
 
-    def calculate
-      return CATEGORY_FAMILY if family_member?
+    def calculate(consider_family: true)
```

Comment:
```suggestion
    def calculate(concern_sac_family: true)
```
consider ist für mich jetzt eher überlegen ob ich was machen soll oder nicht.
concerning ist berücksichtigen auf deutsch, oder?!
ich weiss, das war so spezifiziert :wink: 

ggf. einfach auch `for_sac_family`

Reply:
habs auf `for_sac_family`